Help Desk/helpDesk_createDepartmentProcess.php

```diff
@@ -58,6 +60,15 @@
             if ($departmentID === false) {
                 throw new PDOException('Could not insert group.');
             }
+            $HelpdeskPermissionsGateway = $container->get(HelpdeskPermissionsGateway::class);
```

Comment:
Bit of a nitpick, but this variable should start with a lowercase h. Seems an uppercase H is used in most of the files.